### PR TITLE
Fixed argument parsing for audin.

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: channels/audin/client/audin_main.c [hunk 1]
```diff
@@ -38,13 +38,13 @@
...
-#define MSG_SNDIN_OPEN          0x03
-#define MSG_SNDIN_OPEN_REPLY    0x04
-#define MSG_SNDIN_DATA_INCOMING 0x05
-#define MSG_SNDIN_DATA          0x06
-#define MSG_SNDIN_FORMATCHANGE  0x07
+#define MSG_SNDIN_VERSION		0x01
+#define MSG_SNDIN_FORMATS		0x02
+#define MSG_SNDIN_OPEN			0x03
+#define MSG_SNDIN_OPEN_REPLY	0x04
+#define MSG_SNDIN_DATA_INCOMING	0x05
```
Comment (mfleisz) [inline channels/audin/client/audin_main.c]: This looks wrong ...

**Comment by mfleisz** [comment]
+1

**Comment by bmiklautz** [comment]
@akallabeth can you quickly add a short description how to test.

**Comment by akallabeth** [comment]
@bmiklautz Easiest way is ```/microphone:sys:alsa``` ```/microphone:sys:pulse``` and ```/microphone:sys:oss``` to load the different backends. Should print out which one is in use.

**Comment by bmiklautz** [comment]
+1

**Pull request merged**